OSDOCS: CNV-40769 - Correcting SCSI type in LUN disk sharing doc (#76003)
openshift/openshift-docs: modules/virt-configuring-disk-sharing-lun.adoc

[id="virt-configuring-disk-sharing-lun_{context}"]
= Configuring disk sharing by using LUN

> To protect data on your VM from outside access, you can enable SCSI persistent reservation and configure a LUN-backed virtual machine disk to be shared among multiple virtual machines. Enabling the shared option allows you to use advanced SCSI commands, such as those required for a Windows failover clustering implementation, for managing the underlying storage.

Reviewer: Per ISG,

> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.

```suggestion
To secure data on your VM from unauthorized access, you can enable SCSI persistent reservation and configure a LUN-backed virtual machine disk to be shared among multiple virtual machines. By enabling the shared option, you can use advanced SCSI commands, such as those required for a Windows failover clustering implementation, for managing the underlying storage.
```

Author: I'd say that "enabling option X allows you to do Y" is not really an instance of anthropomorphism, but your suggested rewrite definitely is an improvement. Thanks Shubha!

> When a storage volume is configured as the `LUN` disk type, a VM can use the volume as a Logical Unit Number (LUN) device. As a result, the VM can deploy and manage the disk by using SCSI commands.

Reviewer: ```suggestion
When a storage volume is configured as the `LUN` disk type, a VM can use the volume as a logical unit number (LUN) device. As a result, the VM can deploy and manage the disk by using the SCSI commands.
```

Author: Is there a specific set of SCSI commands that the doc is talking about here? If not, then I don't think the definite article is really warranted here. On the other hand, you're right that capitalizing LUN is probably unnecessary.

> You reserve a LUN through the SCSI persistent reserve options. To enable the reservation, configure the feature gate option and activate it on the LUN disk to issue SCSI device-specific input and output controls (IOCTLs) that the VM requires.

Reviewer: Suggestion: You can split the 2nd sentence into 2 or may be list them out in bullets.

If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
* The storage provider must support a Container Storage Interface (CSI) driver that uses the SCSI protocol.
* If you are a cluster administrator and intend to configure disk sharing by using LUN, you must enable the cluster's feature gate on the `HyperConverged` custom resource (CR).

> * Any disk that you want to share must be in block mode.

Reviewer: Hope this sounds technically correct

```suggestion
* The shared disks must be in block mode.
```

Author: That could imply that the disk is already shared, but setting up the sharing is the point of the procedure, so that wording might be confusing. I'll try to come up with a good compromise.
